.travis.yml

```diff
@@ -118,11 +118,6 @@ deploy:
   provider: releases
   skip_cleanup: true
 
-cache: cargo
-before_cache:
-  # Travis can't cache files that are not readable by "others"
```

Comment:
Lets replace this with a comment explaining why we cannot cache right now, and that once we are back on stable, we should be able to re-add this.